### PR TITLE
[Custom DC] Fix various issues with re-runnign install_custom_dc.sh

### PR DESCRIPTION
**Pull request by Fructokinase**
- Fix provider dependency issue: Fetch kubeconfig in install_custom_dc.sh first
- In Cloud shell, IPv6 sometimes can't be resolved -> add to /etc/hosts manually
- Terraform plan inconsistency issues during re-run -> fetch mixer_grpc_pb outside of terraform
- Make mixer_grpc_pb versioned.

### CODE REVIEW

File: deploy/terraform-datacommons-website/examples/website_v1/main.tf [hunk 1]
```diff
@@ -93,33 +94,21 @@ resource "google_compute_managed_ssl_certificate" "dc_website_cert" {
...
-  location = var.region
-  project = var.project_id
-
-  depends_on = [module.cluster]
-}
-
-data "google_client_config" "default" {}
-
+# IMPORTANT NOTE: This script assumes that
+# "~/.kube/config" already exists. This is because provider cannot depend on data or resources,
```
Comment (shifucun) [inline deploy/terraform-datacommons-website/examples/website_v1/main.tf:98]: Maybe add some comments about what is "kube/config"?

File: deploy/terraform-datacommons-website/examples/website_v1/main.tf [hunk 2]
```diff
@@ -66,6 +66,7 @@ module "apikeys" {
 module "esp" {
   source                   =  "../../modules/esp"
   project_id               = var.project_id
+  mixer_githash            = var.mixer_githash
```
Comment (shifucun) [inline deploy/terraform-datacommons-website/examples/website_v1/main.tf:69]: is website_gitbash needed?

File: scripts/install_custom_dc.sh [hunk 3]
```diff
@@ -14,7 +14,14 @@
...
 set -e
 
-CUSTOM_DC_RELEASE_TAG=custom-dc-v0.1.0
+CUSTOM_DC_RELEASE_TAG=custom-dc-v0.2.0
+
+sudo chmod a+w /etc/hosts
+export APIS="googleapis.com www.googleapis.com storage.googleapis.com iam.googleapis.com container.googleapis.com cloudresourcemanager.googleapis.com"
+for i in $APIS
+do
+    echo "199.36.153.10 $i" >> /etc/hosts
```
Comment (shifucun) [inline scripts/install_custom_dc.sh:23]: what is this used for?

File: scripts/install_custom_dc.sh [hunk 4]
```diff
@@ -14,7 +14,14 @@
 # limitations under the License.
 set -e
 
-CUSTOM_DC_RELEASE_TAG=custom-dc-v0.1.0
+CUSTOM_DC_RELEASE_TAG=custom-dc-v0.2.0
```
Comment (shifucun) [inline scripts/install_custom_dc.sh:17]: Maybe update the tag after it's created?

**Review by shifucun — APPROVED** [review]

File: deploy/terraform-datacommons-website/examples/website_v1/main.tf (hunk 2 above)
Comment (Fructokinase) [reply deploy/terraform-datacommons-website/examples/website_v1/main.tf:69]: nope, only mixer_githash is needed to fetch the versioned mixer grpc pb.

File: deploy/terraform-datacommons-website/examples/website_v1/main.tf (hunk 1 above)
Comment (Fructokinase) [reply deploy/terraform-datacommons-website/examples/website_v1/main.tf:98]: done.

File: scripts/install_custom_dc.sh (hunk 4 above)
Comment (Fructokinase) [reply scripts/install_custom_dc.sh:17]: currently custom-dc-v0.1.0 is used for the previous version of the script, if we update it after, we are changing the script for custom-dc-v0.1.0.

File: scripts/install_custom_dc.sh (hunk 3 above)
Comment (Fructokinase) [reply scripts/install_custom_dc.sh:23]: added a comment.

**Review by shifucun — APPROVED** [review]

**Pull request merged**